src/features/projectsV2/ProjectDetails/components/MultiPlantLocationInfo.tsx

```diff
@@ -17,7 +18,7 @@ import ImageSlider from './ImageSlider';
 import MobileInfoSwiper from '../../MobileInfoSwiper';
 
 interface Props {
-  plantLocationInfo: PlantLocationMulti | undefined;
+  plantLocationInfo: PlantLocationMulti | OtherInterventions;
   isMobile: boolean;
```

Comment:
This probably isn't needed due to #2605

Also, `OtherInterventions` should not be accepted as this component is to display `PlantLocationMulti` type interventions only.